recipes/pkgconf/all/conanfile.py

```diff
@@ -92,9 +96,14 @@ def package(self):
         os.rename(os.path.join(self.package_folder, "share", "aclocal"),
                   os.path.join(self.package_folder, "bin", "aclocal"))
         tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
-        self.cpp_info.includedirs.append(os.path.join("include", "libpkgconf"))
```

Comment:
Was this include directory wrong?

Reply:
I guess so, because [all the source code i could find](https://github.com/pkgconf/pkgconf/search?l=C&q=libpkgconf.h) use use `#include <libpkgconf/libpkgconf.h>`